.github/workflows/build.yml

```diff
@@ -10,11 +10,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6.x
+        ruby-version: 2.6
```

Comment:
Looks like these can be dropped now and it will read the `.ruby-version` https://github.com/ruby/setup-ruby#supported-version-syntax

Reply:
True. However, in .ruby-version we must specify the full 2.6.6 version — since rbenv requires a full version (it can’t work with 2.6). 

I think it’s better to leave the major version in our workflows, which will automatically use the latest 2.6 patch.

Folks building locally with rbenv will still use the .ruby-version — and so I’ve matched that for now.

Downside of that approach is that it repeats ourselves, kinda. But I think it’s better to automatically get patch releases in our workflows. What do you think?

Reply:
Fair enough. I think 2.7.1 is the latest for the pages setup though. I usually peak over at https://github.com/github/pages-gem/ when I'm looking to bump that

Reply:
Yeah. And 2.7 will run a little faster. Tests do pass running locally with 2.7, so that looks okay.

Thinking about this a little more, since we have multiple workflows here, why don’t we leave the ruby version specification to .ruby-version. That way we have just one file to keep up to date, rather than 4 workflows, too.

So I’ll push that commit in a second here. You okay with that approach?

Reply:
Yup, that works for me. I think I came across somewhere that had an automated way to pull down the info from an API to keep it in sync, but I don't recall were now

Reply:
Looks like you cleaned up the other files, but not this one 😉

Reply:
Whoops. Fixed now